src/js/view/preview.js

```diff
@@ -4,15 +4,16 @@ define([
 
     var Preview = function(_model) {
         this.model = _model;
-
-        this.model.on('change:playlistItem', this.loadImage, this);
     };
 
     _.extend(Preview.prototype, {
         setup: function(element) {
             this.el = element;
 
-            this.loadImage(this.model, this.model.get('playlistItem'));
+            this.model.on('change:playlistItem', this.loadImage, this);
```

Comment:
Do we insure that setup is only called once? It might be useful to have a shortcut for `off(event, fn, context).on('event, fn, context)` to avoid duplicate listeners. Maybe that could be part of the `watch` helper I suggested below: